### PR TITLE
RTS (jakarta): WFLY-15447 and WFLY-15545

### PR DESCRIPTION
**Pull request by mmusgrov**
Signed-off-by: Michael Musgrove <mmusgrov@redhat.com>

Produces a jakarta variant of the WildFly RTS subsystem (using source transformation).

Jira issues:

https://issues.redhat.com/browse/WFLY-15447
https://issues.redhat.com/browse/WFLY-15545

### CODE REVIEW

File: ee-9/pom.xml [hunk 1]
```diff
@@ -1536,6 +1536,54 @@
...
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.jboss.narayana.rts</groupId>
+                <artifactId>restat-api-jakarta</artifactId>
```
Comment (yersan) [inline ee-9/pom.xml:1553]: We have a license entry for the nonnative one on `ee-feature-pack-common-licenses.xml`, so I guess we need a new entry for the Jakarta native one.

**Review by yersan — COMMENTED** [review]
I found very minor details, once fixed the changes look good to me

File: ee-9/pom.xml [hunk 2]
```diff
@@ -1536,6 +1536,54 @@
...
+                    <exclusion>
+                        <groupId>*</groupId>
+                        <artifactId>*</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.jboss.narayana.rts</groupId>
+                <artifactId>restat-integration-jakarta</artifactId>
```
[the remark this answers is not in the document]
Comment (mmusgrov) [reply ee-9/pom.xml:1577]: @yersan Do you mean that I need to modify `./ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml` by adding entries like:

```
  <dependency> 
      <groupId>org.jboss.narayana.rts</groupId>
      <artifactId>restat-integration-jakarta</artifactId>
      <licenses>
        <license>
          <name>GNU Lesser General Public License v2.1 only</name>
          <url>http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html</url>
          <distribution>repo</distribution>
        </license>
      </licenses>
    </dependency>
```
and similarly for `restat-api-jakarta` and `restat-bridge-jakarta`?

Or do you mean the file `./ee-feature-pack/common/src/main/resources/license/ee-feature-pack-common-licenses.xml` or some other license file?

**Comment by mmusgrov** [comment]
@yersan the latest push adds the restat-xxx-jakarta entries to `ee-9/common/src/main/resources/license/preview-feature-pack-common-licenses.xml` and the removes the extraneous code comment.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @mmusgrov

**Pull request merged**